Extensions/ArtifactEngine/E2ETests/E2E.ts

```diff
@@ -47,11 +50,76 @@ describe('e2e tests', () => {
                     assert.equal(data, "dummyFolderContent");
                     done();
                 });
-                
+
                 assert.equal(tickets.find(x => x.artifactItem.path == "Extensions/ArtifactEngine/TestData/Jenkins/file1.pdb").retryCount, 0);
                 assert.equal(tickets.find(x => x.artifactItem.path == "Extensions/ArtifactEngine/TestData/Jenkins/folder1/file2.txt").retryCount, 0);
             }, (error) => {
                 throw "test failure";
             });
     });
+
+    it('should be able to download build artifact', function (done) {
+        this.timeout(15000);
+        let processor = new engine.ArtifactEngine();
+
+        let processorOptions = new engine.ArtifactEngineOptions();
+        processorOptions.fileProcessingTimeoutInMinutes = 5;
```

Comment:
fileProcessingTimeoutInMinutes is not used by artifactEngine. can you confirm with Omesh & remove it if this is the case?

Reply:
removed